### PR TITLE
Issue #199 added Object array to LiveGraphFactory to stop throwing ex…

### PR DESCRIPTION
**Pull request by dmmiller612**
There are some design choices I made that could easily be changed. One for example, if you look at convertToObjectArray, I added the private function to handle primitive and Object arrays, which you may only want Object arrays.

### CODE REVIEW

**Comment by bartoszwalacik** [comment]
thnx, will take a look at it tomorrow

**Comment by bartoszwalacik** [comment]
could you give mie write access to https://github.com/dmmiller612/javers? I'd like to put some small changes in this PR

**Comment by dmmiller612** [comment]
You should have write access, now.

File: javers-core/src/main/java/org/javers/core/graph/LiveGraphFactory.java [hunk 1]
```diff
@@ -44,6 +45,10 @@ private Object wrapTopLevelContainer(Object handle){
         if (handle instanceof  Set){
             return new SetWrapper((Set)handle);
         }
+        if (handle.getClass().isArray()){
+            //return new ArrayWrapper(convertToObjectArray(handle));
+            return new ArrayWrapper((Object[])handle);
```
Comment (bartoszwalacik) [inline javers-core/src/main/java/org/javers/core/graph/LiveGraphFactory.java]: @dmmiller612, I've removed this conversion and the tests are still passing, not sure what is it for ...
If this version is ok for you, I can merge

Comment (bartoszwalacik) [inline javers-core/src/main/java/org/javers/core/graph/LiveGraphFactory.java]: @dmmiller612, I've removed this conversion and the tests are still passing, not sure what is it for ...
If this version is ok for you, I can merge

**Comment by dmmiller612** [comment]
The rason I added the conversion is because you can't directly cast Object[] to a primitive[] at least in Java. I don't know much about groovy, and if it boxes the integer to an object. If you are fine just using objects, which is a valid design choice, I am okay with it.

**Comment by bartoszwalacik** [comment]
you are right, seems copying is needed as there is no obvious way to do casting. So I've restored it.
I've added more test cases to cover both primitive and object arrays.

**Pull request merged**